docs/content/en/schemas/v2beta7.json

```diff
@@ -861,6 +866,7 @@
         "helm",
         "kubectl",
         "kustomize",
```

Comment:
Add kpt before kubectl or before kustomize?

Reply:
This schema is auto-generated from the corresponding structure in config.go.  What matters more is the order in which they are handled in [getDeployer()](https://github.com/felixtran39/skaffold/blob/df42484b7c66635847c7bfc429c8ab2f9a360f10/pkg/skaffold/runner/new.go#L190-L217).  I think it makes sense to have kpt be last, since that way it has a chance to act upon the results of the other deployers?

Reply:
The order they are handled in `getDeployer()` shouldn't affect the result either. If alphabetical order is the convention I can change that.

Reply:
oh, what I mean is that strategically we'd like to recommend kpt over kustomize and kubectl to users. Since this specific field is "preferredOrder", we can let kpt comes up before the other less recommended ones. Agree that the order doesn't affect the real config operations. My comment is more from the user's perspective.